ppr-ui/src/views/newRegistration/LengthTrust.vue

```diff
@@ -131,16 +138,38 @@ export default class LengthTrust extends Vue {
                 'Please note that this must be within the last 21 days. The length of the Lien is automatically set ' +
                 'to 180 days.'
       case APIRegistrationTypes.MARRIAGE_MH:
+        return (
```

Comment:
Steph, can you add a change to the repairer's lien text to add a space after the end of the first sentence: change '...surrendered.' +
to
'...surrendered. ' +
This is the last ux assurance change for RL.

Reply:
done

Reply:
Thanks.